data/migrations/deb/x_x_xx_to_y_y_yy.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3.5
+import os
+import shutil
+import glob
+
+from stp_core.common.log import getlogger
+from sovrin_common.config_util import getConfig
+
+config = getConfig()
+logger = getlogger()
+
+old_base_dir = '~/.indy'
+
+def migrate_genesis_txn(old_base_txn_dir, new_base_txn_dir):
+    for suffix in ('sandbox', 'live', 'local'):
+        old_domain_genesis = os.path.join(
+            old_base_txn_dir, 'domain_transactions_{}_genesis'.format(suffix))
+        old_pool_genesis = os.path.join(
+            old_base_txn_dir, 'pool_transactions_{}_genesis'.format(suffix))
+
+        if os.path.exists(old_domain_genesis):
+            shutil.move(old_domain_genesis, new_base_txn_dir)
```

Comment:
It once mentioned that they do not wanna do anything disruptive like moving, they wanna do copying instead. Please write an email to clarify

Reply:
I checked this point with Alexander earlier and we decided to use 'move' strategy instead of 'copy'. Who is the person for clarification? Can I leave as is for now?